scripts/tools_assure_gogenerate.sh

```diff
@@ -5,7 +5,7 @@ cd $GOPATH/src/github.com/raintank/metrictank/
 # stringer needs all packages to be installed to work
 # see https://github.com/golang/go/issues/10249
 #go install ./...
-go install github.com/raintank/metrictank/vendor/github.com/hashicorp/memberlist github.com/raintank/metrictank/vendor/github.com/dgryski/go-tsz 
+govendor go install github.com/hashicorp/memberlist github.com/dgryski/go-tsz 
```

Comment:
this should just be `govendor install github.com/hashicorp/memberlist github.com/dgryski/go-tsz`